src/DocBlock/Tags/See.php

```diff
@@ -56,6 +56,10 @@ public static function create(
         $parts       = preg_split('/\s+/Su', $body, 2);
         $description = isset($parts[1]) ? $descriptionFactory->create($parts[1], $context) : null;
 
+        if (preg_match('{^https?://}', $parts[0])) {
+            return null;
```

Comment:
Only supporting `https?://` seems very limiting. What about people that omit the protocol?
Imo it should simply try to parse it as an FQSEN, and if it is not a valid FQSEN, return something like `Link` instance